src/corppa/ocr/gvision_ocr.py

```diff
@@ -7,60 +7,168 @@
  - install python client: `pip install google-cloud-vision`
 """
 
-import glob
-import io
 import os
-
-from google.cloud import vision
-
-
-IMAGE_DIR = "images"
-
-
-def get_image_ocr():
+import sys
+import io
+import pathlib
+import argparse
+
+from tqdm import tqdm
+
+# Attempt to import Google Cloud Vision Python Client
+try:
+    from google.cloud import vision
+except ImportError:
+    print(
+        "Error: Python environment does not contain google-cloud-vision "
+        "package. Switch environments or install package and try again."
+    )
+    sys.exit(1)
+
+
+def image_relpath_generator(image_dir, ext_set, follow_symlinks=True):
+    for dirpath, dirs, files in image_dir.walk(follow_symlinks=follow_symlinks):
+        # Check the files in walked directory
+        for file in files:
+            ext = os.path.splitext(file)[1]
```

Comment:
I think there's probably a pathlib way to do this. [suffix](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.suffix) maybe?

Reply:
Not in this case. `file` is a string, so I'd have to create an additional pathlib.Path object in order to use pathlib which seemed excessive.